packages/core/src/ceramic.ts

```diff
@@ -381,7 +381,7 @@ export class Ceramic implements CeramicApi {
 
       if (
         (networkOptions.name == Networks.MAINNET || networkOptions.name == Networks.ELP) &&
-        anchorServiceUrl != DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name]
+        !(anchorServiceUrl in [DEFAULT_ANCHOR_SERVICE_URLS[networkOptions.name], "https://cas-internal.3boxlabs.com"])
```

Comment:
why do we use "https://cas-internal.3boxlabs.com" and not "https://cas.3boxlabs.com", which is already in the default URLs map?

Reply:
Bc `cas.3boxlabs` has a firewall that requires a static ip and our machines don't have a static ip that we can add there. So we use an internal load balancer instead which is accessible only by our nodes